dts/arm/nxp/nxp_lpc55S3x_common.dtsi

```diff
@@ -227,6 +227,21 @@
 		status = "okay";
 	};
 
+	adc0: adc@A0000 {
+		compatible = "nxp,lpc-lpadc";
+		reg = <0xA0000 0x1000>;
+		interrupts = <22 0>;
+		status = "disabled";
+		clk-divider = <8>;
+		clk-source = <0>;
+		voltage-ref= <2>;
+		calibration-average = <128>;
+		power-level = <1>;
+		offset-value-a = <10>;
+		offset-value-b = <10>;
+		#io-channel-cells = <1>;
+	};
+
```

Comment:
there is also a second LPADC called `adc1` on this chip, can we add that here

Reply:
Hi @decsny , I was looking into trying to get ADC1 working since I want to make use of it. I was wondering if this is still something planned and if it would help if I can share the part I think that needs to be in there in a seperate PR. It took me some time and Im not sure if I did it right.